graph.c

```diff
@@ -119,7 +119,7 @@ nodepath(struct node *n, bool escape)
 	escape = false;
 	nquote = 0;
 	for (s = n->path->s; *s; ++s) {
-		if (!isalnum(*s) && !strchr("_+-./", *s))
+		if (!isalnum((int)(*s)) && !strchr("_+-./", *s))
```

Comment:
The parens around `*s` aren't necessary

Reply:
and ?

Reply:
And they should be removed, because redundant parens next to casts aren't used elsewhere in the code.

Reply:
let's wait for @michaelforney 's answer

Reply:
Yes, please remove the unnecessary parentheses to be consistent with the rest of the code.